File: Python/Pattern_c1.py
"""
Patetrn

1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4

n = 4

"""

print("Enter the number of rows")
n=int(input())

print("Here is the pattern")

for i in range(1,n+1):
    for j in range(1,n+1):
        print(j,end=" ")
    print("\n")

File: patternsJAVA/patternC1.java
/*
Pattern is: 
1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4
*/
public class patternC1 {
    public static void main(String[] args) {
        int n=4;
        for (int i = 1; i <=n; i++) {
            for (int j = 1; j <=n; j++) {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
